src/trans/gpu/algor/external/fourier/hicfft.h

```diff
@@ -41,7 +41,7 @@ inline void __fftSafeCall(hipfftResult err, const char *file, const int line)
     if( hipSuccess != (int) err) {
         _printError("GPU runtime", file, line, err, _fftGetErrorEnum(err));
         hipDeviceReset();
-        return;
+        abort();
```

Comment:
Would there be an interest in using fiat's c interface to abor1 here?

Reply:
Is that essentially the same as `ABOR1`?

Reply:
https://github.com/ecmwf-ifs/fiat/blob/main/src/fiat/include/fiat/abor1.h#L24

```
#include "abor1.h"

ABOR1("your message goes here");
```